core/GlobalState.h

```diff
@@ -54,9 +54,8 @@ class GlobalState final {
     void initEmpty();
     void installIntrinsics();
 
-    // Expand tables to use approximate `kb` KiB of memory. Can be used prior to
-    // operation to avoid table resizes.
-    void reserveMemory(u4 kb);
+    // Expand tables to the given sizes. Does nothing if the value is <= current capacity.
+    void preallocateTables(u4 symbolSize, u4 nameSize);
```

Comment:
nit: since these `u4` are unitless. would you mind either amending the comment or changing the variable names to suggest that these values measure "length" of the symbol / name table vector, and not bytes of memory?

Reply:
Good idea; fixed in comment.